### PR TITLE
Update Miniconda version on Windows from 4.6.14 to 4.8.3/latest

### PR DESCRIPTION
**Pull request by weiji14**
# Description
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**

Updates miniconda on Windows from 4.6.13 to 4.8.3 (actually to 'latest') to make it in line with the macOS and Ubuntu versions.

## Check list
- [x] Related issue / work item is attached 
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/Windows2019-Readme.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 - NPM 6.14.6
 - Yarn 1.22.4
 - pip 20.2.2 (python 3.7)
-- Miniconda 4.6.14
+- Miniconda 4.8.3
```
Comment (maxim-lobanov) [inline images/win/Windows2019-Readme.md:32]: documentation files (*.md) are auto-generated and updated automatically during deployment. Please revert them

Comment (weiji14) [reply images/win/Windows2019-Readme.md:32]: Ok, will do.

**Review by maxim-lobanov — APPROVED** [review]

**Comment by maxim-lobanov** [comment]
Will try to get more context about 
```
# Lock to Miniconda 4.6 until we do the work to run `conda init` for the vsts user
# Then we can go back to installing the latest Miniconda
```

**Comment by maxim-lobanov** [comment]
cc: @brcrista hello Brian! I have found that some time ago, you have hardcoded Miniconda 4.6.14 by https://github.com/microsoft/azure-pipelines-image-generation/commit/ffc77a7776aaba6e1c96a2d2d1deeef1ca11d50a
Related PR: https://github.com/microsoft/azure-pipelines-image-generation/pull/1151

Do you remember any context around it?
We would like to return back to approach of installing latest version but would like to understand if it can break any pipelines and how we can validate it.

**Comment by brcrista** [comment]
@maxim-lobanov yeah, we used to install latest but then we hit an issue as noted in the comment.  I think what happens is that you just can't activate environments anymore because `conda` will complain that you haven't run `conda init`.

That's a step we would have to add during image generation.  Should be pretty easy.  It just adds some lines to the shell config, so we would have to do it for Bash, PowerShell, and CMD (on Windows).  Then, we can go back to latest on all OSes.

**Review by al-cheb — APPROVED** [review]

**Comment by maxim-lobanov** [comment]
@brcrista , I have deployed the image based on current changes (without `conda init` in image-generation) and run canary tests. They passed successfully.
Any concerns to merge this one? Any other tests that we need to run?

**Review by brcrista — APPROVED** [review]

**Pull request merged**